### ISSUE TITLE
Elasticsearch-fluentd-kibana Helm chart

### ISSUE THREAD

**Issue by andrzejwp** [fluent/fluentd-docs#508]
Hello, I wanted to let you know that we're working on a Helm chart that makes it easier to deploy a Kubernetes logging solution based on fluentd, es and kibana. I'm not sure where this would fit in your docs, maybe in [this article](https://docs.fluentd.org/v0.12/articles/kubernetes-fluentd) ?

The Helm chart is here: https://github.com/cdwv/efk-stack-helm

**Comment by cosmo0920** [comment]
Currently, some k8s package manager exist including helm.
There are some package manager: [ksonnet](https://ksonnet.io/), [kubecfg](https://github.com/ksonnet/kubecfg).
Which should we k8s package manager?

**Comment by andrzejwp** [comment]
@cosmo0920 not sure if I understand what question you're asking here...

**Comment by cosmo0920** [comment]
Is helm standard package manager for k8s?
If so, it looks good to add description to construct EFK stack with k8s helm.

**Comment by andrzejwp** [comment]
Yes, it is quite mainstream. It recently got promoted as an incubation-level project inside CNCF. https://www.cncf.io/blog/2018/06/01/cncf-to-host-helm/

**Comment by cosmo0920** [comment]
I think that we should put kubernetes-fluentd.txt under [docs/v1.0](https://github.com/fluent/fluentd-docs/tree/master/docs/v1.0).
[docs/v0.12/kubernetes-fluentd.txt](https://github.com/fluent/fluentd-docs/blob/master/docs/v0.12/kubernetes-fluentd.txt) is for old stable Fluentd.
We can start to write fluentd for kubernetes article which is based on k8s helm from scratch. :memo:
